README.md

```diff
@@ -13,7 +13,7 @@ High level process
 3. Run the new OpenShift ASP.NET base image
 
 For more information about using these images with OpenShift, please see
-the official OpenShift Documentation (https://docs.openshift.org/latest/using_images/s2i_images/php.html) .
+the official OpenShift Documentation (https://docs.openshift.com/enterprise/3.1/creating_images/s2i.html) .
```

Comment:
@jawnsy The origin docs will have the most up to date information but may contain features that are not available in the enterprise version. I tend to prefer referencing enterprise docs